js/service.js

```diff
@@ -314,12 +316,22 @@ let Service = {
       return getDataFromURL(`${PULSE_API}/v2/myprofile/`);
     },
     put: function(updatedProfile) {
+      if (updatedProfile.newsletter) {
```

Comment:
we may want to do this only if we know the value changed from "not checked" to "checked". The way it is shouldn't cause any issues, but it might post signups that won't do anything when someone updates their profile...

Reply:
Gotcha but is there a way for us to verify user's newsletter subscription state? 🤔

Reply:
yeah that's a good point, I don't think there is right now =(